types/culori/src/interpolate/Interpolator.d.ts

```diff
@@ -1 +0,0 @@
-export type Interpolator = (arr: number[]) => (t: number) => number;
```

Comment:
This type was just flattened, only because this file doesn’t exist in culori, and `Interpolator` had multiple conflicting definitions